src/current/releases/cloud.md

```diff
@@ -15,6 +15,14 @@ Get future release notes emailed to you:
 {% include marketo.html formId=1083 %}
 
 
+## November 1, 2024
+
+Cockroach Labs has announced [updated pricing](https://www.cockroachlabs.com/blog/improved-cockroachdb-cloud-pricing/) for CockroachDB Cloud. This new pricing model goes into effect December 1, 2024 for new customers and existing pay-as-you-go customers, and upon renewal for annual or multi-year contract customers.
```

Comment:
If you take some of the feedback above, it would be good to reflect it here too.